### ISSUE TITLE
Not working in Thunderbird 60

### ISSUE THREAD

**Issue by jasmines** [jamtur01/lattedock-thunderbird#1]
Ubuntu 18.04, Thunderbird 60, Latte Dock 0.8.1, extension 0.0.3

**Comment by jamtur01** [comment]
Yes - I am aware but I've had no time to debug. Help welcomed.

**Comment by jasmines** [comment]
Here you are:

NS_ERROR_XPC_BAD_IID: Component returned failure code: 0x80570018 (NS_ERROR_XPC_BAD_IID) [nsIJSCID.createInstance] lattedockunread.js:47
	updateUnreadCount chrome://lattedockunread/content/lattedockunread.js:47:14
	performUnreadCount chrome://lattedockunread/content/lattedockunread.js:82:3
	onItemCountChanged/this.timeoutId< chrome://lattedockunread/content/lattedockunread.js:65:51
	openModalWindow jar:file:///usr/lib/thunderbird/omni.ja!/components/nsPrompter.js:353:5
	openPrompt jar:file:///usr/lib/thunderbird/omni.ja!/components/nsPrompter.js:563:9
	confirmEx jar:file:///usr/lib/thunderbird/omni.ja!/components/nsPrompter.js:704:9
	openModalWindow jar:file:///usr/lib/thunderbird/omni.ja!/components/nsPrompter.js:353:5
	openPrompt jar:file:///usr/lib/thunderbird/omni.ja!/components/nsPrompter.js:563:9
	confirmEx jar:file:///usr/lib/thunderbird/omni.ja!/components/nsPrompter.js:704:9

**Comment by jamtur01** [comment]
@jasmines Can you try the build in dist/ and let me know if you see any errors and/or it works.

**Comment by jasmines** [comment]
@jamtur01 it works perfectly! Thanks a lot!

**Comment by jasmines** [comment]
Unfortunately, it works on a PC and not on another. No errors in the debug... I'm investigating...

**Comment by jamtur01** [comment]
Any difference between the PCs?

**Comment by jasmines** [comment]
I'm trying to figure it out...

**Comment by jamtur01** [comment]
Do you have AppArmor? Can you check for any AppArmor errors in Syslog?

**Comment by jasmines** [comment]
Yes, I have it but no errors are found in syslog.
A little bit of info, more:

basically the counter works as I start Thunderbird, but suddenly it stops working, I can't say what event is fired before this happens.
I have some extensions installed, I'm wondering if someone could be conflicting...

Calendar Tweaks
Classic Password Editor
ExQuilla for Microsoft Exchange
gContactSync
Invia Dopo
Lightning
LightningButton
Provider per Google Calendar
SMTP Switch Reborn
TbSync
Thunderbird Conversations
ThunderKeepPlus
ThunderPEC

**Comment by jasmines** [comment]
Now it's saying it's not compatible with Thunderbird version 68.1.2
I tried to change the value in install.rdf, no luck

**Comment by jamtur01** [comment]
Yeah I've given up maintaining this I am afraid. The API is just too fragile. :(

**Comment by jasmines** [comment]
NOOOOOOOOOOOO :(
